rfcs/protocols/asset-transfer/corda.md

```diff
@@ -6,9 +6,19 @@
 # Asset Transfer Protocol Units in Corda Networks
 
 - RFC: 02-010
-- Authors: Venkatraman Ramakrishna, Sandeep Nishad, Krishnasuri Narayanam, Dhinakaran Vinayagamurthy
+- Authors: Krishnasuri Narayanam, Sandeep Nishad, Venkatraman Ramakrishna, Dhinakaran Vinayagamurthy
 - Status: Proposed
-- Since: 01-Jun-2021
+- Since: 21-Oct-2022
 
-## Summary
+## Weaver Asset Transfer implementation approach for Corda network
```

Comment:
Avoid using terms like "approach" in the protocol specs. That's be more appropriate in a research paper where we are looking for peer validation. Here, we are declaring what we are doing in the Weaver software project. I'd suggest renaming the title to just "Implementation of Protocol Units in Weaver" ("Corda" is implied by the filename, or you can retain the Summary section and write an introductory line.)

Reply:
Done